### PR TITLE
Ukrainian localization

### PR DESCRIPTION
**Pull request by MeDustyy**

### CODE REVIEW

**Review by Boxadactle — APPROVED** [review]

**Review by Boxadactle — APPROVED** [review]

**Comment by MeDustyy** [comment]
@Boxadactle you approved this PR but did not merge it, please do so 🙃

**Comment by Boxadactle** [comment]
Actually, I have merged it into the dev branch, so it will not show as merged until it gets published and merged into the main branch.

**Pull request merged**
